### ISSUE TITLE
Binding snooze time to snooze option

### ISSUE THREAD

**Issue by zebulun78** [Windos/BurntToast#71]
I am having trouble figuring this one out.  I see this is supposed to be able to be bound but with a lack of example I have been unable to figure out just how to do that.  I have a dropdown with 4 time options, and want to bind the output to the snooze option.

Cheers!

**Comment by zebulun78** [comment]
I think I figured it out.

$SnoozeOption1 = New-BTSelectionBoxItem -Id '60' -Content '1 Hour'

This is how it seems to work.  You pass the minutes in the Id.  This was not clear in the documentation.  It would be great if you added some examples that included of the Submit-BTNotification scenarios.

**Comment by Windos** [comment]
Yeah, sorry about this, documentation is going to be my big focus for the next few weeks to months covering scenarios like this in better (or ANY in some cases) detail.

If you're still having issues with this in general let me know and I can probably do a blog post on it specifically.

**Comment by RichardDee** [comment]
I am not zebulun78, but I would be happy if you do a blog post on it specifically, because I thought that you cant retrieve the selected data from a SelectionBox due to not being able to subscribe to WinRT events in PowerShell as said in " https://github.com/Windos/BurntToast/issues/26 ".

**Comment by Windos** [comment]
Sorry, yeah I'll get a blog post done on this topic... have bumped it up my list. If it's not the next one it'll be the one after.

In general though, the snooze thing is probably the one and only area where selection boxes work. That's because it's the system/API itself using the selection, not PowerShell.

**Comment by RichardDee** [comment]
Thats fine enough! I use the BurntToast Module to pop a toastnotification as soon as the computer needs a reboot (to ensure proper software installations). The first button is already created and executes a Restart.exe which is defined within the registry and the second button should postpone the notication and the snooze function would be a valid solution to re-pop the notification in x minutes/hours.
